bin/git-forgit

```diff
@@ -867,7 +908,19 @@ export FORGIT_GI_REPO_LOCAL="${FORGIT_GI_REPO_LOCAL:-${XDG_CACHE_HOME:-$HOME/.ca
 export FORGIT_GI_TEMPLATES=${FORGIT_GI_TEMPLATES:-$FORGIT_GI_REPO_LOCAL/templates}
 
 _forgit_ignore_preview() {
-    $_forgit_ignore_pager "$FORGIT_GI_TEMPLATES/$1"{,.gitignore} 2>/dev/null
+    files=()
+    files=( "$FORGIT_GI_TEMPLATES/$1"{,.gitignore} )
+    if [[ -n $FORGIT_IGNORE_PAGER ]]; then
+        escaped_files=()
+        for file in "${files[@]}"; do
+            escaped_files+=("'$file'")
+        done
+        eval "$FORGIT_IGNORE_PAGER ${escaped_files[*]}" 2> /dev/null
+    elif hash bat &> /dev/null; then
+        bat -l gitignore --color=always "${files[@]}" 2>/dev/null
+    else
+        cat "${files[@]}" 2> /dev/null
+    fi
```

Comment:
just checking this change was intentional - more of a change than the rest of the files.

Reply:
Intentional, yes but not final. I will try to slim it down in case we decide to go this route. The difference with this one is that we end up with two paths in the `$files` array due to the `{,.gitignore}` expansion. When using `eval`, both should be quoted separately (hence the extra for loop). Also we have two fallbacks here (bat and cat) that we do not use anywhere else.